transformers/test_data/vat_flux.go

```diff
@@ -53,20 +53,17 @@ var VatFluxHeaderSyncLog = core.HeaderSyncLog{
 	Transformed: false,
 }
 
-var VatFluxModel = shared.InsertionModel{
+var VatFluxModel = event.InsertionModel{
```

Comment:
I think we want to make the actual `event.InsertionModel` private and export a `VatFluxModel()` function that calls `CopyEventModel` (so that we don't modify properties on the var directly.

Reply:
Ok.. I think I'll need to do this in a few other places as well.